.gitignore

```diff
@@ -8,6 +8,7 @@ release.properties
 .project
 .settings
 .vscode
+.factorypath
```

Comment:
This seems fine. (Though I would typically just put it in `~/.gitignore` since it is a dumb aspect of Eclipse JDT-based tools, not really specific to any repo.)